{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "192a0156_f5a04733",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1012732
      },
      "writtenOn": "2020-11-20T22:11:57Z",
      "side": 1,
      "message": "CI\u0027s flaky gcloud66-78537567b1d6 again here failed this check. Re-running now, hoping that another node will run it.",
      "revId": "7bcd79e1835003f48a1d2456207503d44c13376c",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    }
  ]
}